plugins/gatsby-transformer-home-example-code/gatsby-node.js

```diff
@@ -3,7 +3,6 @@
  */
 
 const crypto = require('crypto');
-const path = require('path');
```

Comment:
This was just an unused var, reported on the lint test, so I've decided to remove it.
This was the error:
```6:7  warning  'path' is assigned a value but never used. Allowed unused vars must match /^_/  no-unused-vars```